code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm

```diff
@@ -38,7 +38,7 @@
 						ABNORMALITY_WORK_ATTACHMENT = list(50, 50, 55, 55, 60),
 						ABNORMALITY_WORK_REPRESSION = list(20, 20, 20, 0, 0)
 						)
-	work_damage_amount = 7
+	work_damage_amount = 15
```

Comment:
The fact that her damage DOUBLES when she goes hysteric (which is a regular occurrence) has NOT been accounted for, causing her do deal **30** black damage per tick. Please consider reducing this to 10-12